pkg/lib/journal.js

```diff
@@ -230,7 +234,34 @@
                 };
             if (count > 1)
                 parts['count'] = count;
-            if (prio < 4)
+            if (ident === 'abrt-notification') {
+                // Problem can be identified by 3 elements. Most likely the first, but in some
+                // cases other two can be used as well.
+                var all_p = [entry['PROBLEM_DIR'], entry['PROBLEM_DUPHASH'], entry['PROBLEM_UUID']];
+                var p;
+                var known = false;
+                for (var i = 0; i < all_p.length; i++) {
+                    if (all_p[i] in displayable_problems) {
+                        p = all_p[i];
+                        known = true;
+                        break;
+                    }
+                }
+                //Do not render unknown (deleted) problems
+                if (! known)
+                    return "";
```

Comment:
So you think that if the same problem happens more than once, we should display first, second and all other occurrences? It may however look a bit weird when you filter out only problems and there will be the same line several times.

I cannot agree with you about the count though. I think it is quite an important information. You will definitely pay more attention if you see that something crashed several times.